Added correct live ranking brackets

Swackles opened the pull request:
I added in the correct live league brackets. Not 100% sure about platinum and diamond leagues, as I got these values from the in-game leaderboard.

I also made the league image larger to improve visibility.

Swackles committed:
feat: Added correct live ranking brackets

Swackles committed:
fix: Made league images larger for improved visibility

leonlarsson commented:
I have been referring to https://www.reachthefinals.com/leaderboard for the different league points and that seems to be accurate to my current version. I'll try and double check the points

leonlarsson committed:
test commit

leonlarsson commented:
It looks like their official leaderboard does not match the game. One could strongly argue that the game should be followed, but I'll try to convince them that their own leaderboard is inconsistent. I'll also try to get the 100% intended values. For now, I'll keep the old values.

I like your other changes, so I'll merge those soon. Thanks! :)

leonlarsson committed:
revert new league values (comment in PR)

Apply Prettier.
Minor changes.

I will try to confirm the intended league values.

Swackles commented:
I'm guessing the ppl who made the web weren't made aware of the league changes and displays leagues based on the open beta. I don't see it likely they made that chance by accident , especially since it hasn't been reverted yet.

I belive that game values should be taken as the basis for rankings and can say that everything up to Platinum 4 was 100% correct. After that, I didn't have access to real data.
